foundationdb/datadog_checks/foundationdb/config_models/validators.py

```diff
@@ -0,0 +1,13 @@
+# (C) Datadog, Inc. 2022-present
```

Comment:
```suggestion
# (C) Datadog, Inc. 2021-present
```
nit- it was added in 2021: https://github.com/DataDog/integrations-extras/commits/fc5555e173900013b1bf95ce749959f43b22346e/foundationdb/datadog_checks/foundationdb/config_models/validators.py

Reply:
I left it as 2022 since this file is new. Or should I go with the date that the integration was added in 2021?

Reply:
Oh strange it showed up as added in the commit history but not in the commit, that's fine then!

Reply:
Oh you're right, the file was there, but it was empty. I'll fix it